src/status_im/ui/screens/intro/styles.cljs

```diff
@@ -4,31 +4,98 @@
 
 (def intro-view
   {:flex               1
+   :justify-content    :flex-end
    :padding-horizontal 30})
 
 (def intro-logo-container
-  {:flex            1
-   :align-items     :center
+  {:align-items     :center
    :justify-content :center})
 
+(defn dot-selector [n]
+  {:flex-direction  :row
+   :justify-content :space-between
+   :align-items     :center
+   :height          6
+   :width           (+ 6 (* (+ 6 10) (dec n)))})
```

Comment:
Magic?

Reply:
Added surrounding let giving meaningful names to numbers 6 and 10.